h/assets.py

```diff
@@ -2,9 +2,41 @@
 import re
 
 from deform.field import Field
+from webassets.filter import ExternalTool, register_filter
 import pyramid
 
 
+class Browserify(ExternalTool):
+    """
+    An input filter for webassets that browserifies CoffeeScript or JavaScript.
+
+    The browserify command-line client has several limitations when piping
```

Comment:
Webassets filters are provided with a python file handle, `in_`, which you're expected to use as input to your program. If we cheat and read off the disk using `source_path, then the browserify filter can only be used as the first filter in the chain, which seemed messy to me. The only other alternative is writing out tempfiles, which some filters do, but again browserify is a pain -- it would have to be a tempfile within the repository otherwise config from `package.json` wouldn't be picked up. This seemed like the least bad option.